docs/guides/connecting_to_your_data/cloud/gcs/pandas.md

```diff
@@ -2,4 +2,231 @@
+```
+
+Load your DataContext into memory using the `get_context()` method.
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/pandas/inferred_and_runtime_yaml_example.py#L6
+```
+
+### 3. Configure your Datasource
+
+Before showing any configurations, we believe it is important to discuss the following:
+
+#### Differences between DataConnectors
+
+Great Expectations provides two types of `DataConnectors` classes for connecting to GCS: `InferredAssetGCSDataConnector` and `ConfiguredAssetGCSDataConnector`
+
+  - An `InferredAssetGCSDataConnector` utilizes regular expressions to infer `data_asset_names` by evaluating filename patterns that exist in your bucket. This `DataConnector`, along with a `RuntimeDataConnector`, is provided as a default when utilizing our Jupyter Notebooks.
+  - A `ConfiguredAssetGCSDataConnector` requires an explicit listing of each `DataAsset` you want to connect to. This allows for more granularity and control than its `Inferred` counterpart but also requires a more complex setup.
+
+As the `InferredAssetDataConnectors` have fewer options and are generally simpler to use, we recommend starting with them.
+
+We've detailed example configurations for both options in the next section for your reference.
+
+#### Authentication / gcs_options
+
+It is also important to note that GCS `DataConnectors` support various methods of authentication. You should be aware of the following options when configuring your own environment:
+* `gcloud` command line tool / `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
+  - This is the default option and what is used throughout this guide.
+* Passing a `filename` argument to the optional `gcs_options` dictionary.
+  - This argument should contain a specific filepath that leads to your credentials JSON.
+  - This method utilizes `google.oauth2.service_account.Credentials.from_service_account_file` under the hood.
+* Passing an `info` argument to the optional `gcs_options` dictionary.
+  - This argument should contain the actual JSON data from your credentials file in the form of a string. 
+  - This method utilizes `google.oauth2.service_account.Credentials.from_service_account_info` under the hood.
+
+Please note that if you use the `filename` or `info` options, you must supply these options to any GE objects that interact with GCS (i.e. `PandasExecutionEngine`). 
+The `gcs_options` dictionary is also responsible for storing any `**kwargs` you wish to pass to the GCS `storage.Client()` connection object (i.e. `project`)
+
+Using these example configurations, add in your GCS bucket and path to a directory that contains some of your data:
+<Tabs
+  groupId="inferred-or-configured"
+  defaultValue='inferred'
+  values={[
+  {label: 'Inferred', value:'inferred'},
+  {label: 'Configured', value:'configured'},
+  ]}>
+
```

Comment:
Hmm I definitely agree but since we went through great lengths to describe the differences between Inferred and Configured in the previous section, what are your thoughts on `Inferred + Runtime (Default)`? Too cumbersome of a title?

Reply:
After seeing it rendered, I like it :)